Docs(output) : Explictily Add info about deafult values (#3579)
webpack/webpack.js.org: src/content/configuration/output.md

  }
};
```

> T> The dafault value of `output.ecmaVersion` in webpack 4 is `5`.

Reviewer: ```suggestion

T> The dafault value of `output.ecmaVersion` in webpack 4 is `5`.
```

> T> Deafult ecmaVersion is 2015(6) for webpack5 and for webpack4 it is 5.

Reviewer 2: ```suggestion
T> Default ecmaVersion is 2015(6) for webpack5 and for webpack4 it is 5.
```

> T> Deafult ecmaVersion is 2015(6) for webpack5 and for webpack4 it is 5.

Reviewer: When we reference specific webpack version, the format is following:
- webpack 4
- webpack 5

It is also mentioned in [contribution guide](https://webpack.js.org/contribute/writers-guide/#typesetting)

We mention types and default after option title, remove the info about webpack 5 as the docs are for webpack 5 explicitly and only keep the warning mentioning default for webpack 4

```suggestion
T> The dafault value of `output.ecmaVersion` in webpack 4 is `5`.
```
